src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  display: block;
+  margin: 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.form-big {
+  margin: 20px 0;
+}
+
+.form {
+  box-sizing: border-box;
+  position: relative;
+}
+
+.big {
+  width: 100%;
+}
+
+.big:before {
+  display: block;
+  position: absolute;
+  content: "";
+  background-image: url(./images/search.svg);
+  background-repeat: no-repeat;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar-big {
+  height: 70px;
+  padding: 23px 0 23px 62px;
+  font-size: 16px;
+}
+
+.search-bar {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  width: 100%;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar:hover {
+  box-sizing: border-box;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-bar:focus {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
why do we need to change a lot of styles on focus? should we do it?

Reply:
In the task it was specified to use styles from "Figma" . I copied this from "Figma". I deleted lines from 80 to 83, like you previous comment. And everything works.